bundles/org.openhab.binding.omnikinverter/src/main/resources/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="omnikinverter" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
```

Comment:
Please use the OH schema URLs instead of the ESH schema URLs:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<binding:binding id="omnikinverter" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
```